packages/scaffold-config/src/frameworks.ts

```diff
@@ -87,7 +93,10 @@ export const FRONTEND_FRAMEWORKS = [
       module.exports = {
         component: {
           devServer,
-        }
+          devServerConfig: {
+            indexHtmlFile: 'cypress/support/component-index.html',
```

Comment:
If this is the default value, I think I’d prefer it to not be in the template. I’ve been told that whatever we can do (in reason) to minimize the amount of setup in the config file should be done